### DISCUSSION TITLE
"LiveCam" enhancements released!

### DISCUSSION THREAD

**Discussion by noodnik2** [SAHorowitz/MSFS2020-PilotPathRecorder#9] [Show and tell]
Hi Steve,

FYI - I finally got around to [publishing](https://github.com/noodnik2/MSFS2020-PilotPathRecorder/releases/latest) the ["LiveCam" enhancements](https://github.com/noodnik2/MSFS2020-PilotPathRecorder/blob/master/README-kmlcam.md) that [we discussed previously](https://github.com/SAHorowitz/MSFS2020-PilotPathRecorder/discussions/4).

I'll invite you and others to check out some [demo videos](https://www.youtube.com/playlist?list=PLWILAAihTU0H0pjbA2KU-NqgNaBqMTE7_) I recently recorded to get a better idea of what it's all about.

Thanks again for the great base on which to build!

-- Marty

**Comment by SAHorowitz** [comment]
Marty,
  That's awesome.  I will definitely check it out.  It may be a few weeks as I am in the process of moving which is way more work than I wish it to be.

I should also be publishing an updated version with some new features people asked for as well as some rewriting to support auto-retry for simconnect.  Be sure to check out the code when it hits.

Glad you are enjoying the software and congrats on the LiveCam feature.

Steve
